tests/installation/leap2sle_zdup.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Those are steps for leap to sle zypper dup. Includes: switch to a tty, full patch
+# leap system, install SUSEConnect, Register at SCC to get SLE repo, List and disable all
+# openSUSE repo, add modules need for installation, Migrate installed packages to SLES repo,
+# Remove orphaned packages, reboot the system
+# Maintainer: Yutao Wang <yuwang@suse.com>
+
+use base "installbasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use registration;
+use version_utils 'is_desktop_installed';
+use power_action_utils 'power_action';
+
+sub run {
+    my ($self) = @_;
+
+    $self->wait_boot(textmode => !is_desktop_installed(), bootloader_time => 300, ready_time => 600);
+
+    select_console('root-console');
+    # Create a snapshot with specified description to do snapper rollback
+    if (get_var('HDDVERSION', '') !~ /13.1/) {
```

Comment:
Why hardcode 13.1 here? We should get it from setting variable.

Reply:
This is copied from zdup old code which used by zdup migration. This is the only version needed at here.

Reply:
Hmm, I wonder if we need this condition at all, as leap 13.1 is officially not supported for a long time. Do we still want to test zdup for those images?

Reply:
Migration team currently only focus on leap15.2 related check.  I think it's safe to remove in my view.